kenshin-profile-app/kenshin-profile-app/Hobby.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Pastime.swift
+//  kenshin-profile-app
+//
+//  Created by miyamotokenshin on R 7/06/09.
+//
+
+import SwiftUI
+
+struct Pastime: View {
```

Comment:
Viewだから最後にViewをつけたほうがいいね

```suggestion
struct PastimeView: View {
```

Reply:
済